packages/cfd/src/Containers/dmt5-trade-modal.tsx

```diff
@@ -185,17 +203,17 @@ const DMT5TradeModal = observer(
                     </div>
                 </div>
                 <div className='cfd-trade-modal__download-center-app'>
-                    <div className='cfd-trade-modal__download-center-app--option'>
+                    <div className='cfd-trade-modal__download-center-app--option' onClick={mobileURLSet}>
                         <Icon icon='IcRebrandingMt5Logo' size={32} />
                         <Text className='cfd-trade-modal__download-center-app--option-item' size='xs'>
                             {localize('MetaTrader 5 web')}
                         </Text>
                         <a
                             className='dc-btn cfd-trade-modal__download-center-app--option-link'
                             type='button'
-                            href={mt5_trade_account.webtrader_url}
-                            target='_blank'
-                            rel='noopener noreferrer'
+                            href={is_mobile ? url : mt5_trade_account.webtrader_url}
```

Comment:
I think it would be less confusing to use `is_mobile` consistently
```JavaScript
<a
  className="dc-btn cfd-trade-modal__download-center-app--option-link"
  type='button'
  href={is_mobile ? url : mt5_trade_account.webtrader_url}
  target={is_mobile ? '' : '_blank'}
  rel={is_mobile ? '' : 'noopener noreferrer'}
>
```

Reply:
Done